Dockerfile

```diff
@@ -0,0 +1,48 @@
+#!/bin/echo docker build . -f
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: MPL-2.0
+#{ 
+# Copyright 2018-present Samsung Electronics France SAS, and other contributors
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.*
+#}
+
+FROM ubuntu:18.04
```

Comment:
Wouldn't it make more sense to start with one of the [node base images](https://hub.docker.com/_/node/)?

Reply:
Yes this doable too, but I made this file originally for iotjs and decided to split patch for node too.